app/main.ipynb

```diff
@@ -31,17 +32,52 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
-    "# write your code for task 0 here"
+    "df = pd.read_csv(\"../data/adult.csv\")\n",
+    "df.replace(\"?\", pd.NA, inplace=True)\n",
+    "df.dropna(inplace=True)\n",
+    "\n",
+    "df[\"salary\"] = df[\"salary\"].str.replace(\"K\", \"\", regex=False).astype(str)\n",
+    "df[\"salary K$\"] = pd.to_numeric(df[\"salary K$\"], errors=\"coerce\")\n",
+    "df.dropna(subset=[\"salary\", \"salary K$\"], inplace=True)\n",
+    "df.isna().sum()"
```

Comment:
you should check data for correctness. So if salary = "<=50K", then you should ensure that column "salary K$" has an appropriate value
also incorrect output